### PR TITLE
Fixes the CI

### PR DESCRIPTION
**Pull request by kelson42**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -31,7 +31,7 @@ jobs:
     - name: installing dependencies
       run: npm ci
       env:
-        PHANTOMJS_CDNURL: http://cnpmjs.org/downloads
+        PHANTOMJS_CDNURL: https://dev.kiwix.org/mwoffliner/phantomjs
```
Comment (kelvinhammond) [inline .github/workflows/ci.yml:34]: Will this work? https://cdn.jsdelivr.net/npm/phantomjs@2.1.3/

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -9,11 +9,11 @@ on:
 jobs:
   build:
 
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
 
     strategy:
       matrix:
-        node-version: [14.x, 16.x, 17.x]
+        node-version: [14.x, 16.x, 18.x]
```
Comment (kelvinhammond) [inline .github/workflows/ci.yml:16]:
```suggestion
        node-version: [14.x, 16.x, 17.x, 18.x]
```
Nodejs 17 is the current version. Is this being skipped?

**Review by kelvinhammond — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (kelson42) [reply .github/workflows/ci.yml:34]: It does, because otherwise the CI would not be green :) right ?

File: .github/workflows/ci.yml (hunk 2 above)
Comment (kelson42) [reply .github/workflows/ci.yml:16]: `18.3.0` is current. https://nodejs.org/en/

Comment (kelvinhammond) [reply .github/workflows/ci.yml:16]: Right you are, outdated link on my end.

**Review by kelvinhammond — APPROVED** [review]
Ship it.

**Pull request merged**